content/community/community-team/community-building-roles/contents.lr

```diff
@@ -13,54 +13,71 @@ Community building roles are for folks interested in helping grow CC's open sour
 If you'd like to apply for one of these roles, please see the main [Community Team][communityteam] page.
 
 ## Community Contributor
-**Who should apply:** If you’ve helped anyone else in the CC open source community on GitHub, Slack, or email, you should apply for this role!
+**Who should apply:**  
+If you’ve helped anyone else in the CC open source community on GitHub, Slack, or email, you should apply for this role!
 
-**What does this role give you?**
-* You’ll receive invites to the private CC Community Team Slack channel and email list.
+**What does this role give you?**  
 * You’ll receive previews of upcoming changes to the Community Team program.
 * You’ll be listed as [a member of the Community Team on CC Open Source](/community/community-team/members/).
 
+**How can you engage with the community?**  
+You can use the following channels to engage with the community:
+* [Vote access](/community/community-team/engagement-channels/#vote-access)
```

Comment:
Everyone in the community team has regular access to the tweets doc since it's in the community team shared drive.

Reply:
Updated the access to reflect that. Now we have "Edit" (everything that can be performed in the Google Doc) and "Publish" (access to Twitter & TweetDeck).